lib/trento_web/controllers/v1/installation_controller.ex

```diff
@@ -24,7 +24,7 @@ defmodule TrentoWeb.V1.InstallationController do
 
   @spec get_api_key(Plug.Conn.t(), any) :: Plug.Conn.t()
   def get_api_key(conn, _) do
-    key = Trento.Infrastructure.Installation.get_api_key()
+    key = Trento.Settings.get_api_key()
```

Comment:
I am wondering if it would make sense to move this `get_api_key` action to the `SettingsController`.

Reply:
Done!